config.js

```diff
@@ -17,6 +17,7 @@ module.exports = {
       , 'es6/Reflect': 'bower_components/harmony-reflect/reflect'
       , 'codemirror': 'bower_components/codemirror'
       , 'jszip': 'bower_components/jszip/dist/jszip'
+      , 'filesaver': 'bower_components/file-saver.js/FileSaver'
```

Comment:
See my last comment, it's all in there: https://github.com/metapolator/metapolator/pull/411#issuecomment-94572896 :-)